src/codegen/julia.jl

```diff
@@ -274,8 +274,10 @@ function emit_leaf_call(cmd::LeafCommand, ctx::EmitContext, args::Symbol, kwargs
         for option in values(cmd.options)
             Configurations.is_option(option.type) || continue
             push!(ret.args, quote
-                $kwargs[$(QuoteNode(option.sym))] =
-                    $Configurations.from_dict($(option.type), $kwargs[$(QuoteNode(option.sym))])
+                if $(QuoteNode(option.sym)) in Base.keys($kwargs)
```

Comment:
```suggestion
                if $(QuoteNode(option.sym)) in $Base.keys($kwargs)
```

althought we have access to `Base` most of the time, but this won't work if it's a `baremodule` which does not have access to `Base`, so we need to insert the module value at compile time.